TransformationSystem/Client/TransformationClient.py

```diff
@@ -2,24 +2,16 @@
 
```

Comment:
This is mostly removing unnecessary hollow methods. I do not think there is a change in the interface